content/docs/gke/authentication.md

```diff
@@ -7,15 +7,89 @@ weight = 4
 
 ## In-cluster authentication
 
-When you [set up Kubeflow for GCP](/docs/gke/deploy), it will automatically 
+When you [set up Kubeflow for GCP](/docs/gke/deploy), it will automatically
 [provision three service accounts](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/#gcp-service-accounts) with different
-privileges. In particular, the `${KFAPP}-user` service account is meant to grant your user services access to GCP. 
+privileges. In particular, the `${KFAPP}-user` service account is meant to grant your user services access to GCP.
 The credentials to this service account can be accessed within the cluster as a
 [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/).
 
 The secret will have basic access to a limited set of GCP services by default, but more roles can be granted through the
 [GCP IAM console](https://console.cloud.google.com/iam-admin/).
 
+### Namespaces
+
+Starting from Kubeflow version 0.6, you can consume Kubeflow in individual namespaces. The `kubeflow` namespace is
+intended for running Kubeflow system components while individual jobs and model deployments are expected to run in
+separate namespaces. In order to do this you will need to install GCP credentials into the new namespace and create
+a PodDefault object to attach the credentials to certain pods.
+
+##### Credentials
+
+You can add credentials to the new namespace by either copying them from an existing Kubeflow namespace or by
+creating a new service account.
+
+To copy credentials from one namespace to another namespace use the following CLI commands (**Note**: there is an
+[issue](https://github.com/kubeflow/kubeflow/issues/3640) filed to automate these commands):
+
+```
+NAMESPACE=<new kubeflow namespace>
+SOURCE=kubeflow
+NAME=user-gcp-sa
```

Comment:
Is this the right name? It was my understanding that the secret name is now ${KFAPP}-user

Reply:
@DanSanche the K8s secret should still be user-gcp-sa
https://github.com/kubeflow/kubeflow/blob/1c07de4d4825f381062c7d7091398b837113b7f1/bootstrap/cmd/bootstrap/app/saKey.go#L152

Reply:
See below and
https://github.com/kubeflow/kubeflow/blob/1c07de4d4825f381062c7d7091398b837113b7f1/bootstrap/cmd/bootstrap/app/saKey.go#L152

This should be correct